benches/coprocessor_executors/mod.rs

```diff
@@ -44,7 +45,8 @@ fn bench_table_scan_next(
                 meta.clone(),
                 ranges.to_vec(),
                 store.to_fixture_store(),
-                false,
+                CountCollectorDisabled,
```

Comment:
I don't think `XXXDisabled` is a good naming. It sounds like a bool variable.

Reply:
Indeed it is intended to act like a bool variable.